### DISCUSSION TITLE
[gatsby-source-strapi] Is v3 compatible with Gatsby v5?

### DISCUSSION THREAD

**Discussion by AkimaLunar** [gatsby-uc/plugins#456] [Q&A]
We have recently upgraded a few dependencies and started seeing missing fields in queries, where the field in a component is pointing to another component. We have upgraded the following:

- "@strapi/strapi": "4.5.3" >> "4.13.2"
- "gatsby-source-strapi": "^2.0.0" >> "^3.3.1"
- "gatsby": "^4.24.6" >> : "^5.12.4"

The docs [here](https://www.gatsbyjs.com/plugins/gatsby-source-strapi/) say gatsby-source-strapi is compatible with Gatsby v4 (not v3). Should it also work with v5?

---

To add more context, we are seeing a lot of errors like `Cannot query field "x" on type "STRAPI__COMPONENT_X"`, which we didn’t have before. The data is Strapi and those field are actually populated, but they are not coming through the source plugin and not showing up in the graph.

**Comment by moonmeister** [comment]
Not sure what's up. I don't personally maintain this plugin. Sounds like some work needs to be done. PRs would be welcome.
